src/Testcontainers.FirebirdSql/FirebirdSqlContainer.cs

```diff
@@ -0,0 +1,62 @@
+namespace Testcontainers.FirebirdSql;
+
+/// <inheritdoc cref="DockerContainer" />
+[PublicAPI]
+public sealed class FirebirdSqlContainer : DockerContainer, IDatabaseContainer
+{
+    private readonly FirebirdSqlConfiguration _configuration;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FirebirdSqlContainer" /> class.
+    /// </summary>
+    /// <param name="configuration">The container configuration.</param>
+    /// <param name="logger">The logger.</param>
+    public FirebirdSqlContainer(FirebirdSqlConfiguration configuration, ILogger logger)
+        : base(configuration, logger)
+    {
+        _configuration = configuration;
+    }
+
+    /// <summary>
+    /// Gets the FirebirdSql connection string.
+    /// </summary>
+    /// <returns>The FirebirdSql connection string.</returns>
+    public string GetConnectionString()
+    {
+        string database;
+
+        if (_configuration.Image.Tag.StartsWith("2.5") || _configuration.Image.Tag.StartsWith("v2.5"))
+        {
+            database = string.Join("/", [_configuration.Environments.TryGetValue("DBPATH", out var dbPath) && !string.IsNullOrEmpty(dbPath) ? dbPath : "/firebird/data", _configuration.Database]);
+        }
+        else
+        {
+            database = _configuration.Database;
+        }
```

Comment:
Ah, it is likely that this is something the user can override. The default value is probably `/firebird/data`. Additionally, I think that Java's implementation reads the value from the running container, while .NET does not. `_configuration` holds the information to start the container, not the information about the actual running container.

Reply:
Or does the configuration (connection string ) depent on the actual client? Maybe the absolute path is only necessary for the Java client?

Reply:
Yes, the Java version reads from the containers's ENV. DBPATH is also default set to /firebird/data. I think it is not necessary here, so the way it is left out now is good.

Reply:
Did you see my latest changes? I moved it to the configuration class.

Reply:
I did, and I think it is a good place for that logic to be.

Reply:
> Yes, the Java version reads from the containers's ENV. DBPATH is also default set to /firebird/data. I think it is not necessary here, so the way it is left out now is good.

Ok, then I think we can remove the part related to `DBPATH` since it does not read the variable from the container.

Reply:
Ah, sorry - it is already addressed and outdated.